src/HeapDump/CrossGenerationLiveness/CrossGenerationLivenessCollector.cs

```diff
@@ -223,19 +224,28 @@ public static void LoadNative(string relativePath)
         /// <summary>
         /// Get the name of the architecture of the current process.
         /// </summary>
-        private static string ProcessArch
+        private static ProcessorArchitecture ProcessArch
         {
             get
             {
-                if (s_ProcessArch == null)
+                return Environment.Is64BitProcess ? ProcessorArchitecture.Amd64 : ProcessorArchitecture.X86;
```

Comment:
❓ Do we need to support ARM here? I figured no since HeapDump only works on AMD64 or X86.